lambdas/importer.ts

```diff
@@ -7,7 +7,7 @@ async function fetchEvents(meetupAccessToken: string) {
 	const meetupGraphQlEndpoint = 'https://api.meetup.com/gql';
 	const batchSize = 10; // Number of events to fetch in each batch
 
-	const GROUP_URLNAMES = ['open-sgf', 'springfield-women-in-tech', 'sgfdevs'];
+	const GROUP_URLNAMES = process.env['MEETUP_GROUP_URLNAMES']?.split(',');
```

Comment:
Typically I'd just use the dot syntax to access things in `process.env`. Meaning this call would become `process.env.MEETUP_GROUP_URLNAMES`